### DISCUSSION TITLE
Active redstone

### DISCUSSION THREAD

**Discussion by Xermacin** [maruohon/minihud#357] [Ideas]
It would realy help if you ad a new function that let you tipe in a block corrd. if the block is active with redstone or not active, maaybe you also can name it like: 1. 8Name], active/yes (green)/notactive/no(red), so you just click a button and u get a list where you see if its active or not.

**Comment by maruohon** [comment]
I added this to the todo list. If it gets added, then it will be in the mod rewrite versions, not before then.

**Comment by Xermacin** [comment]
how long does it take and pls make it for 1.19.4 too

**Comment by Samalando** [comment]
It's not like coding time, masa has like 500 things on his todo list so it'll take time for it to reach the top

**Reply by Xermacin** [reply]
oh :(
